apps/dashboard/app/controllers/filenavigator_controller.rb

```diff
@@ -0,0 +1,211 @@
+class FilenavigatorController < ApplicationController

```

Comment:
We shouldn't need a new controller. Just some views and utilize the existing FS API.